src/main/java/com/hotels/hcommon/hive/metastore/client/tunnelling/MetastoreTunnel.java

```diff
@@ -97,7 +97,7 @@ public void setStrictHostKeyChecking(String strictHostKeyChecking) {
     this.strictHostKeyChecking = strictHostKeyChecking;
   }
 
-  public boolean isStrictHostKeyChecking() {
+  public boolean getIsStrictHostKeyChecking() {
```

Comment:
A better name can be getStrictHostKeyCheckingFlag() and the variable can be strictHostKeyCheckingFlag.